BAZProjectC1/View/TrendingViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  TrendingViewController.swift
+//  BAZProjectC1
+//
+//
+
+import UIKit
+
+class TrendingViewController: UITableViewController {
+
+    fileprivate var movies: [MovieModel] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(MovieBasicInfoCell.nib(), forCellReuseIdentifier: MovieBasicInfoCell.identifier)
+    }
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        getDataInfo(urlString: EndpointsList.movieAPI.description)
+    }
+    func getDataInfo(urlString: String) {
+        ApiServiceRequest.getService(urlService: urlString, structureType: MovieApiResponseModel.self, handler: { [weak self] dataResponse in
+            if let data = dataResponse as? MovieApiResponseModel {
+                self?.reloadDataToTable(with: data)
+            }
+        })
+    }
+    func reloadDataToTable(with data: MovieApiResponseModel)  {
+        self.movies = data.results
+        self.tableView.reloadData()
+    }
```

Comment:
Ten en cuenta los espacios entre líneas. Considera instalar [swiftlint](https://github.com/realm/SwiftLint)

```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()
        tableView.register(MovieBasicInfoCell.nib(), forCellReuseIdentifier: MovieBasicInfoCell.identifier)
    }
    
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        getDataInfo(urlString: EndpointsList.movieAPI.description)
    }
    
    func getDataInfo(urlString: String) {
        ApiServiceRequest.getService(urlService: urlString, structureType: MovieApiResponseModel.self, handler: { [weak self] dataResponse in
            if let data = dataResponse as? MovieApiResponseModel {
                self?.reloadDataToTable(with: data)
            }
        })
    }
    
    func reloadDataToTable(with data: MovieApiResponseModel) {
        self.movies = data.results
        self.tableView.reloadData()
    }
```